include/git2/worktree.h

```diff
@@ -86,10 +86,15 @@ typedef struct git_worktree_add_options {
 
 	int lock; /**< lock newly created worktree */
 	git_reference *ref; /**< reference to use for the new worktree HEAD */
+
+	/**
+	 * Options for the checkout.
+	 */
+	git_checkout_options checkout_opts;
```

Comment:
We're not super consistent about `opts` vs `options` here, but I'd prefer `options` when a struct is embedded within another struct.

```suggestion
	git_checkout_options checkout_options;
```